games-fps/redeclipse/redeclipse-2.0.0.ebuild

```diff
@@ -0,0 +1,82 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit desktop
+
+DESCRIPTION="First-person ego-shooter, built as a total conversion of Cube Engine 2"
+HOMEPAGE="https://www.redeclipse.net"
+SRC_URI="https://github.com/red-eclipse/base/releases/download/v${PV}/${PN}_${PV}_nix.tar.bz2"
```

Comment:
```
>>> Emerging (1 of 1) games-fps/redeclipse-2.0.0::testworld
>>> Downloading 'https://github.com/red-eclipse/base/releases/download/v2.0.0/redeclipse_2.0.0_nix.tar.bz2'
--2020-07-20 08:48:28--  https://github.com/red-eclipse/base/releases/download/v2.0.0/redeclipse_2.0.0_nix.tar.bz2
Resolving github.com... 140.82.118.3
Connecting to github.com|140.82.118.3|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2020-07-20 08:48:28 ERROR 404: Not Found.

!!! Couldn't download 'redeclipse_2.0.0_nix.tar.bz2'. Aborting.
```

Reply:
Good spot. Fixed, I'm doing a clean `emerge` to double check and I'll push the fix.